pkg/api/defaults-cloud-controller-manager.go

```diff
@@ -5,18 +5,27 @@ package api
 
 import (
 	"strconv"
+	"strings"
 
 	"github.com/Azure/aks-engine/pkg/api/common"
 )
 
 func (cs *ContainerService) setCloudControllerManagerConfig() {
 	o := cs.Properties.OrchestratorProfile
+	isAzureCNIDualStack := cs.Properties.IsAzureCNIDualStack()
+	clusterCidr := o.KubernetesConfig.ClusterSubnet
+	if isAzureCNIDualStack {
+		clusterSubnets := strings.Split(clusterCidr, ",")
+		if len(clusterSubnets) > 1 {
```

Comment:
I think we want `if len(clusterSubnets) > 0` here if the reason is to not index a non-existent array item.

Reply:
sorry, ignore this!